### ISSUE TITLE
Libraries unavailable after sign-in on ATV

### ISSUE THREAD

**Issue by goosen8r** [iBaa/PlexConnect#401]
Works great for local server when not signed in on ATV. After sign-in, I can see and access remote PMS's, but my own will show libraries, but no content available. Restarted/rebooted PMS server and ATV. Signed out and in on ATV and PMS. Do not have PlexPass and have disabled home users on PMS and ATV. Have also tried signing in as home user on ATV and no luck. Additionally my local net is allowed without auth to PMS.

PMS Logs from a connection from ATV: http://pasted.co/06109a15
PlexConnect Logs: http://pasted.co/b4b6b89c

**Comment by iBaa** [comment]
a) When signed in discovery is done via plex.tv only. Publish your local sever.
b) Check the DNS re-bind to local network setting of your router. Add `plex.direct` to your white list.

**Comment by goosen8r** [comment]
I do have the server published, and it is accessible from the plex.tv webapp. I added the plex.direct domain to the rebind whitelist with no success, so I tried disabling rebinding checks all together which still didn't work.  I do have an outside *.com domain and an inside *.int domain, and I use PfSense for DNS resolver, not forwarder as I saw in another post changes were made to the forwarder configuration. Are there any screenshots that would help, as I would like to not have to revert to the old version of PlexConnect I was using.

**Comment by goosen8r** [comment]
I browsed this post: https://forums.plex.tv/discussion/165649/pms-0-9-12-3-cant-see-server-on-ssl/p2
and I see that most times when DNS rebinding is an issue, users are unable to connect securely to their own server when on their network and connecting from plex.tv. This is not what I am experiencing, though I would like to point out that I did have that issue, but resolved simply by putting the external FQDN of my pfSense router into the advanced settings in Plex under the Network tab. Seems Plex.tv may be redirecting my internal devices to that URL instead of plex.direct? I'm lost on this. My ATV is able to see all libraries, just not content; and I stood up another Plex instance on Ubuntu and I have the exact same issue there as well.
